src/lake/Lake/Build/Module.lean

```diff
@@ -181,13 +184,38 @@ def Module.cFacetConfig : ModuleFacetConfig cFacet :=
       -- do content-aware hashing so that we avoid recompiling unchanged C files
       return (mod.cFile, ← fetchFileTrace mod.cFile)
 
+/-- The `ModuleFacetConfig` for the builtin `bcFacet`. -/
+def Module.bcFacetConfig : ModuleFacetConfig bcFacet :=
+  mkFacetJobConfigSmall fun mod => do
+    (← mod.leanArts.fetch).bindSync fun _ _ =>
+      -- do content-aware hashing so that we avoid recompiling unchanged bitcode files
+      return (mod.bcFile, ← fetchFileTrace mod.bcFile)
+
 /-- Recursively build the module's object file from its C file produced by `lean`. -/
-def Module.recBuildLeanO (self : Module) : IndexBuildM (BuildJob FilePath) := do
-  buildLeanO self.name.toString self.oFile (← self.c.fetch) self.weakLeancArgs self.leancArgs
+def Module.recBuildLeanCToO (self : Module) : IndexBuildM (BuildJob FilePath) := do
+  -- TODO: add option to pass target.
```

Comment:
Could you elaborate some more on these TODOs? It need not be in the code itself, but I would like to know at least in this discussion what your hopes are for this in the future.

Reply:
This is about adding an option for cross compilation. As we understood those would need to be fed to the compiler at this point.

Reply:
@hargoniX What is needed to implement this? Is just another configuration option? If so, I would have seen no problem adding it here. But since we are trying to get this PR out the door, a follow-up may be better.